### ISSUE TITLE
[BUG] Sidebar doesn't update all values on logout

### ISSUE THREAD

**Issue by soundstorm** [RocketChat/Rocket.Chat.iOS#1118]
When logging out from one server and switching automatically to another server following params weren't updated:
 - Server Name (retaining old servers title)
 - Profile picture (set to default svg, neither updating to the current servers image nor retaining the old servers image)
 - Online status (offline)
So more or less the whole top elements except the username are not updated correctly in the sidebar.

**Comment by soundstorm** [comment]
Okay, with build 136 following behaviour:

- have at least 2 servers connected
- logout from the server at position 2 (if you log out from the first server it seems to work normally)
- open sidebar:
 • Server name = Label, username = john.appleseed, profile picture = blank

**Comment by soundstorm** [comment]
@rafaelks this seems not to be fixed 🙈
We need to call `SubscriptionsViewController.updateCurrentUserInformation()` somewhere after switching server, not only on `willReveal()`

**Comment by soundstorm** [comment]
I'm maybe just too impatient. The values are all loaded correctly, but async. So if you open the sidebar before we got all the information of the server, the value stays at "john.appleseed" until next time.

Long term: can we store the data like server name and username and load these values instead of waiting until we got them from the server? I've just seen another App showing "Label" for a short time somewhere, but may look unprofessional to some people 😇
Goal: rock solid app 👊
